client/sections.js

```diff
@@ -236,7 +236,7 @@ const sections = [
 		name: 'domains',
 		paths: [ '/domains' ],
 		module: 'calypso/my-sites/domains',
-		group: 'sites',
+		group: 'domains',
```

Comment:
This change was necessary otherwise `section` and `sectionGroup` are always `sites`, making it more difficult to add the active background color to the link.

Let me know if we are against this change, and give me a potential solution 😄

Reply:
An alternative would be to use the section name instead, but we'd have to update the other items, as without this change, the `My Sites` is active even when I am at `/domains/manage`.  

But given `domains` are now top-level, I think it makes sense that they are their own `group`, so it looks fine to me.

Reply:
The other thing is going to do is remove the `.is-group-sites` class from the `<body>` and replace it with `.is-group-domains`. These group classes are used to apply styles just for a particular group of sections in the app.

Afaict we aren't depending on any of the `.is-group-sites` styles so it should be fine to move.

However I'm think we should actually keep `/domains` in the `sites` group for a different reason. I'll put together a video to demonstrate what I mean.